app/konnect/network.md

```diff
@@ -35,14 +35,17 @@ want to use port `3001` for the proxy, map `3001:8000`.
 
 ## Hostnames
 
-Add the following hostnames to the allowlist to give the
-{{site.konnect_short_name}} control plane access through the firewall:
+Data planes initiate the connection to the {{site.konnect_short_name}} control plane.
```

Comment:
In the context of Konnect, we still refer to Runtime instance, not Data plane, right?

Reply:
This brings back the whole runtime instance vs data plane terminology discussion. Let me take that to the docs team and get back to you.

Reply:
Closing the loop: after a chat with the team, agreed that it makes sense to use "runtime instance" in the Konnect context in most situations, including the one you called out here. 

The "data plane" term will still be used, but in a more generic way. Based on that criteria, the "Network Resiliency and Availability" topic remains as-is, using the "data plane" terminology.